docker/kubernetes-tentacle/scripts/configure-and-run.sh

```diff
@@ -230,13 +230,22 @@ function registerTentacle() {
     tentacle "${ARGS[@]}"
 }
 
+function run() {
+    echo "==============================================="
+    echo "Starting Octopus Deploy Kubernetes Tentacle"
+    echo "==============================================="
+
+    exec tentacle agent --instance Tentacle --noninteractive
```

Comment:
the `exec` here means that the host bash process is replaced by the `tentacle` process

Reply:
Maybe add this comment as a comment in the bash script?